.gitignore

```diff
@@ -1,6 +1,18 @@
-node_modules/
-
 # misc
 .DS_Store
 npm-debug.log*
-.env
\ No newline at end of file
+
+# next.js build output
+.next
+
+# dotenv environment variables file (build for Zeit Now)
+.env
+.env.build
+
+# Dependency directories
+node_modules/
+
+# Logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Remove

Reply:
Done but kept .env

Reply:
Already covered by the .gitignore in root

Reply:
Oh, it is the root one, never mind